.github/workflows/runtime_discord_notify.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       actor: ${{ github.event.pull_request.user.login }}
 
   notify:
-    if: ${{ needs.check_maintainer.outputs.is_core_team == 'true' }}
+    if: ${{ needs.check_maintainer.outputs.is_core_team == 'true' && github.event.label.name == 'React Core Team' }}
```

Comment:
What about instead of using the label, we used the PR created event?

```js
on:
  pull_request_target:
    types: [closed]
    paths-ignore:
```

the only reason I used the label before is that we didn't have the core team info in the repo